SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DCPower/Source.cs

```diff
@@ -121,7 +121,34 @@ internal static void ForceVoltage(this DCPowerSessionsBundle sessionsBundle, IDi
         }
 
         /// <summary>
-        /// Forces voltage on the target pins at the specified level. Must at least provide a level value, and the method will assume all other properties that have been previously set.  Optionally, can also provide a specific current limit, current limit range, voltage level range values directly.
+        /// Forces voltage on the target pins at the specified level. Must at least provide a level value, and the method will assume all other properties that have been previously set. Optionally, can also provide a specific current limit, current limit range, voltage level range values directly.
```

Comment:
Suggest changing to: "Forces voltage on the target pins at the specified site-unique level."